api/src/controllers/website.js

```diff
@@ -75,51 +75,28 @@ async function deleteWebPage(req, res, next) {
 }
 
 /**
- * Update publications layout & sorting options
- * @param {*} req request object, containing the teamId and option object
- * @param {*} res response object
- * @returns 200: publication options successfully updated in the DB
- * @returns 500: Server error while saving new page name to DB
+ * Patch client website metadata (title, publication options, layout and/or theme)
+ * @param {*} req request object, an array of objects describing the 'field' to change and it's 'value'
+ * @param {*} res response object (dictionary of changes made)
+ * @returns 200: changes updated in DB
+ * @returns 500: Server error while saving to DB
  */
-async function updatePublicationOptions(req, res, next) { // eslint-disable-line no-unused-vars
-  const updatedPubOptions = req.body;
-  const { teamId: _id } = req.params;// TODO: teamId get from the token instead of parameters?
-  try {
-    await Website.updateOne(
-      { teamId: _id },
-      {
-        $set: {
-          publicationOptions: updatedPubOptions,
-        },
-      },
-    );
-    return res.status(200).json(updatedPubOptions);
-  } catch (err) {
-    return res.send(fillErrorObject(500, 'Server error', [err.errors]));
-  }
-}
+async function patchClientWebMetadata(req, res, next) { // eslint-disable-line no-unused-vars
+  const data = req.body;
+  const { team_id: _id } = req.params;// TODO: teamId get from the token instead of parameters?
```

Comment:
Just to address your comment, I think the team id ends up in the request anyway from the auth middleware when it decodes the jwt?

Reply:
hahaha I think that's @pmyip3 's comment